lib/hanami/view/scope_builder.rb

```diff
@@ -31,7 +31,7 @@ def scope_class(name = nil, rendering:)
           elsif name.is_a?(Class)
             name
           else
-            View.cache.fetch_or_store(:scope_class, rendering.config) do
+            View.cache.fetch_or_store("scope_class: #{name}", rendering.config) do
```

Comment:
:bulb: If it helps, in my local patch, I only use `name` which is all you need to store in the cache. Example:

``` ruby
View.cache.fetch_or_store name, rendering.config do
  # Implementation details.
end
```